pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -461,8 +474,9 @@ func (e *eksOperatorController) deployEKSOperator() error {
 
 	systemProject, err := project.GetSystemProject(localCluster, e.projectCache)
 	if err != nil {
-		panic(err)
```

Comment:
Well goodness, when you said "remove panic" I didn't think it was that literal :)